src/components/slices/project-grid.js

```diff
@@ -51,15 +53,19 @@ export const ProjectGrid = ({ slice }) => {
                       alt=""
                       className={sty.image}
```

Comment:
Because the image is the only thing linking right now we'll want to add alt text or a title to the link. Ideally something like "View *project name*" versus just view project. 
Alternatively you can keep the blank alt tag and make sure that the title or whole card links to the project instead.